examples/analog_out/cont_gen_voltage_wfm_int_clk_non_regen.py

```diff
@@ -0,0 +1,41 @@
+"""Example of analog output voltage generation.
+
+This example demonstrates how to continuously generate an
+analog output waveform by providing new data to the output buffer
+as the task is running.
+
+This example is useful if you want to generate a non-repeating waveform,
+make updates on-the-fly, or generate a frequency that is not an
+even divide-down of your sample clock. In this example,
+the default frequency value is 17.0 to demonstrate that non-regenerative output
+can be used to create a signal with a frequency that is not an even divide-down
+of your sample clock.
+"""
+
+from analog_out_helper import create_sine_wave
+
+import nidaqmx
+from nidaqmx.constants import AcquisitionType, RegenerationMode
+
+
+with nidaqmx.Task() as task:
+    is_first_run = True
+    sampling_rate = 1000.0
```

Comment:
The DAQ device may not be able to output at exactly this sample rate. When that happens, it coerces the sample clock rate property to the closest rate that it can output. You should read back the sample rate in order to avoid introducing frequency error. 

You might think "any DAQ device should be able to generate exactly 1000.0, right?" However, devices with delta-sigma converters may not, depending on their oversample clocks and divisors. Also, the user is free to change the sampling rate.

Reply:
Noted, added `actual_sampling_rate = task.timing.samp_clk_rate` to all impacted example files.